packages/server/graphql/types/GitHubIssue.ts

```diff
@@ -1,55 +1,31 @@
-import {GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GraphQLID, GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
 import connectionDefinitions from '../connectionDefinitions'
 import {GQLContext} from '../graphql'
-import GraphQLISO8601Type from './GraphQLISO8601Type'
 import GraphQLURLType from './GraphQLURLType'
 import PageInfoDateCursor from './PageInfoDateCursor'
 import StandardMutationError from './StandardMutationError'
-import Story, {storyFields} from './Story'
-import ThreadSource from './ThreadSource'
+import GitHubRepository from './GitHubRepository'
 
 const GitHubIssue = new GraphQLObjectType<any, GQLContext>({
   name: 'GitHubIssue',
   description: 'The GitHub Issue that comes direct from GitHub',
-  interfaces: () => [Story, ThreadSource],
-  isTypeOf: ({nameWithOwner}) => !!nameWithOwner,
   fields: () => ({
-    ...storyFields(),
     id: {
       type: GraphQLNonNull(GraphQLID),
       description: 'The id of the issue as found in GitHub'
     },
     url: {
       type: GraphQLNonNull(GraphQLURLType),
       description: 'The url to access the issue'
-      // resolve: ({cloudName, key}) => {
-      //   return `https://${cloudName}.atlassian.net/browse/${key}`
-      // }
     },
-    nameWithOwner: {
-      type: GraphQLNonNull(GraphQLID),
-      description: 'The owner / repo of the issue as found in GitHub'
+    repository: {
```

Comment:
GitHub returns the `repository` which contains the `nameWithOwner`. Perhaps I should update [SuggestedIntegrationGitHub](https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/server/graphql/types/SuggestedIntegrationGitHub.ts) so that it follows this same format?